pyproject.toml

```diff
@@ -36,7 +37,7 @@ select = [
     "INP",
 ]
 
-[tool.ruff.mccabe]
```

Comment:
Shouldn't this say `[tool.ruff.lint.mccabe]` per https://docs.astral.sh/ruff/settings/#lintmccabe?

<img width="976" height="436" alt="Screenshot 2025-12-05 at 10 48 02 AM" src="https://github.com/user-attachments/assets/6073999e-d888-4ea7-8fd8-c85dd32d4f59" />